pythonfile/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM python:3.6.1
+CMD ["python", "first.py"]
```

Comment:
Example taken from the python docker image [hub website](https://hub.docker.com/_/python/):

```
FROM python:3

WORKDIR /usr/src/app

COPY requirements.txt ./
RUN pip install --no-cache-dir -r requirements.txt

COPY . .

CMD [ "python", "./your-daemon-or-script.py" ]
```

By coping you can get the source files into the image (also fixing your windows problem / see slack) and redistribute our whole app as a container.
Also running `pip` will install all dependencies.